src/Http/Error.php

```diff
@@ -55,6 +55,11 @@ public function __construct(
         $lorisInstance = $request->getAttribute('loris');
         $user          = $request->getAttribute('user') ?? new \LORIS\AnonymousUser();
 
+        // Variables used to suggest the user to login and later redirect them if they
+        // are not authenticated in a 403.
+        $tpl_data['anonymous'] = $user instanceof \LORIS\AnonymousUser;
+        $tpl_data['url']       = urlencode($_SERVER['REQUEST_URI']);
```

Comment:
This should be using the URI from the `ServerRequestInterface` passed as an argument, not the PHP superglobal.

Reply:
It's a bug unrelated to your PR, just use $uri->__toString and create an issue about it.

Reply:
I am not sure we should use `$uri->__toString()` before the issue is fixed (if we decide to fix it) as it seems to happen on most setups except for yours and Github Actions' ones. I personally think we should either fix the issue first or add a comment and merge. I'll try to see if there is a clean quick fix for the issue if I have some time today.

I created the issue at #9049.